turbo/services/interfaces.go

```diff
@@ -34,12 +34,24 @@ type HeaderReader interface {
 }
 
 type BorEventReader interface {
+	LastEventId(ctx context.Context, tx kv.Tx) (uint64, bool, error)
 	EventLookup(ctx context.Context, tx kv.Getter, txnHash common.Hash) (uint64, bool, error)
 	EventsByBlock(ctx context.Context, tx kv.Tx, hash common.Hash, blockNum uint64) ([]rlp.RawValue, error)
 }
 
 type BorSpanReader interface {
-	Span(ctx context.Context, tx kv.Getter, spanNum uint64) ([]byte, error)
+	LastSpanId(ctx context.Context, tx kv.Tx) (uint64, bool, error)
```

Comment:
It will be nice if we just pick 1 style and stick to it for consistency - I don't mind either `LastSpanId` or `LastSpanID` but I think myself and @battlmonstr have been following the latter in recent PRs.

Or maybe this is a lost cause and devs will never agree on this... 😂 in which case im happy to just stop paying attention to it

Reply:
ok - I'm happy to change my ways to use Id too